### PR TITLE
Add try/except around Message creation

### PR DESCRIPTION
**Pull request by TAlonglong**
Add try/except around Message creation in request manager loop.

This to avoid the request manager loop to crash if it get an invalid message.

 - [x] Tests added  
 - [x] Fully documented 
 - [x] Add your name to `AUTHORS.md` if not there already

### CODE REVIEW

File: trollmoves/server.py [hunk 1]
```diff
@@ -242,7 +242,12 @@ def _run(self):
                 address, payload = self._get_address_and_payload()
                 if payload is None:
                     continue
-                self._process_request(Message(rawstr=payload), address)
+                try:
+                    self._process_request(Message(rawstr=payload), address)
+                except MessageError:
+                    LOGGER.exception("Failed to create message from payload: %s with address %s",
+                                     str(payload), str(address))
+                    pass
```
Comment (mraspaud) [inline trollmoves/server.py:250]:
```suggestion
```

**Review by mraspaud — COMMENTED** [review]
Could we add a test for this?

**Comment by TAlonglong** [comment]
Working on the test now.

Why not pass? I know its not needed, but it clearly indicates that the intention is not the break the loop

**Comment by mraspaud** [comment]
> Working on the test now.
> 
> Why not pass? I know its not needed, but it clearly indicates that the intention is not the break the loop

`pass` is usually just written to avoid a syntax error, like in
```python
def dummy_function():
    pass
```
It's unusual in python to use it as you do, so I thought it was something that you forgot to clean up. I would still remove it though, as the flow is clear to me without it.

**Comment by TAlonglong** [comment]
Hm, tests passes locally, but not in CI.

Looks like caplog is empty whereas I expected some output. hm

**Comment by TAlonglong** [comment]
yay,

**Review by mraspaud — APPROVED** [review]
LGTM!

**Pull request merged**